src/api-gen/FunctionMarkdownBuilder.php

```diff
@@ -11,9 +11,13 @@ class FunctionMarkdownBuilder {
   private ?DocBlock $docblock;
 
   public function __construct(
-    string $file,
+    ?string $file = null,
```

Comment:
Btw, this doesn't affect me from merging this -- which I am going to do right now :) If we need to change it to be non-nullable, we can do that later.